### PR TITLE
Create Bo's first post with a trinket and markdown formatting

### PR DESCRIPTION
**Pull request by IMBLAH**

### CODE REVIEW

File: _posts/IMBLAH/2021-10-20-IMBLAH-first-trinket-post.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+- - -
```
Comment (sarahva47) [inline _posts/IMBLAH/2021-10-20-IMBLAH-first-trinket-post.md]: remember to do "---" not "- - -"!

Comment (IMBLAH) [reply _posts/IMBLAH/2021-10-20-IMBLAH-first-trinket-post.md]: Thank you so much!

Comment (MishaCienfuegos) [reply _posts/IMBLAH/2021-10-20-IMBLAH-first-trinket-post.md]: Nice catch!

**Review by MishaCienfuegos — APPROVED** [review]
Everything looks spot on!

**Pull request merged**